src/style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+
+}
+
+.bi {
+  width: calc(100% - 62px);
+  height: 70px;
+  border-radius: 4px;
+  border: solid 1px #e1e7ed;
+  padding-left: 62px;
+  margin: 20px auto;
+  box-shadow: 0 1px 8px;
+}
```

Comment:
Some of this styles are same as in `sm` class, DRY, apply them to `inp` class, and override sizes in specific for size classes